html-css/week3/index3.html

```diff
@@ -0,0 +1,123 @@
+              <label for="shoes">Shoes</label>
+              <input type="checkbox" name="shoes" id="shoes" />
+              <label for="handbags">Handbags</label>
+              <input type="checkbox" name="handbags" id="handbags" />
+            </div>
+            <div>
+              <p>Choose the images that best resemble your style</p>
+            </div>
+            <div class="gallery-items">
+              <div>
+                <img src="./images/img1.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img1">Select</label>
+                  <input type="checkbox" name="img1" id="img1" />
+                </div>
+              </div>
+              <div>
+                <img src="./images/img2.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img2">Select</label>
+                  <input type="checkbox" name="img2" id="img2" />
+                </div>
+              </div>
+              <div>
+                <img src="./images/img3.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img3">Select</label>
+                  <input type="checkbox" name="img3" id="img3" />
+                </div>
+              </div>
+              <div>
+                <img src="./images/img4.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img4">Select</label>
+                  <input type="checkbox" name="img4" id="img4" />
+                </div>
+              </div>
+              <div>
+                <img src="./images/img5.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img5">Select</label>
+                  <input type="checkbox" name="img5" id="img5" />
+                </div>
+              </div>
+              <div>
+                <img src="./images/img6.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img6">Select</label>
+                  <input type="checkbox" name="img6" id="img6" />
+                </div>
+              </div>
+              <div>
+                <img src="./images/img7.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img7">Select</label>
+                  <input type="checkbox" name="img7" id="img7" />
+                </div>
+              </div>
+              <div>
+                <img src="./images/img1.jpg" alt="clothes" />
+                <div class="gallery-item">
+                  <label for="img8">Select</label>
+                  <input type="checkbox" name="img8" id="img8" />
+                </div>
+              </div>
+            </div>
+            <div class="footer-box steps">
+              <button>Next step</button>
+            </div>
+          </form>
+        </section>
+      </div>
+    </main>
+    <footer><div class="container"></div></footer>
```

Comment:
`<footer><div class="container"></div></footer>` what's the content of this?

Reply:
Hi Gohar. Thanks for your review. 
It happened because I had one template for all pages, but later changed the structure and didn’t check all pages. Sorry for that.